packages/snjs/lib/protocol/payloads/deltas/remote_conflicts.ts

```diff
@@ -69,8 +142,7 @@ export class DeltaRemoteConflicts extends PayloadsDelta {
         PayloadSource.DecryptedTransient
       );
```

Comment:
(commenting on line 139)

`mostRecent` most likely does not work as intended and is the cause of duplicated tags.